### PR TITLE
Added Best Buy, homebridge, intuit, mailgun, newegg logos 

### PR DESCRIPTION
**Pull request by captchadex**

### CODE REVIEW

File: vectors/ecobee/ecobee.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg version="1.1" id="logo-text" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" style="enable-background:new 0 0 101.58 21.57;" xml:space="preserve" viewBox="Infinity Infinity -Infinity -Infinity">
```
Comment (tijme) [inline vectors/ecobee/ecobee.svg:1]: It looks like this logo is not responsive yet. So it won't show up in Raivo OTP correctly.

Comment (tijme) [reply vectors/ecobee/ecobee.svg:1]: The rest of the icons are okay.

Comment (captchadex) [reply vectors/ecobee/ecobee.svg]: I've deleted ecobee for now. :)

**Comment by tijme** [comment]
Thanks @captchadex,

Just merged your icons. They'll arrive in Raivo in a few minutes. Make sure to tap the refresh button in the top right corner to load the new icons.

**Pull request merged**